contact.html

```diff
@@ -36,6 +36,8 @@
   	<link rel="stylesheet" href="css/vegas.min.css">
     <link rel="stylesheet" href="css/navstyle.css">
 	<link rel="stylesheet" href="css/style.css">
+    <script src="https://kit.fontawesome.com/845b7f6704.js" crossorigin="anonymous"></script>
```

Comment:
This script need to remove

Reply:
@pankajan05 this is for fontawesom icons. Should I use images instead of icons?

Reply:
Yeah, that is fine because it will create unnecessary network load. and you can download the icons and you can use them.

Reply:
Ok got it. All the issues will be resolved by today only